src/XrdHttp/XrdHttpExtHandler.cc

```diff
@@ -28,24 +28,25 @@
 #include "XrdOuc/XrdOucEnv.hh"
 
 /// Sends a basic response. If the length is < 0 then it is calculated internally
-int XrdHttpExtReq::SendSimpleResp(int code, const char* desc, const char* header_to_add, const char* body, long long int bodylen)
+int XrdHttpExtReq::SendSimpleResp(int code, const char* desc, const char* header_to_add, const char* body, long long bodylen)
```

Comment:
This is a new commit which was not here before. Can we just fix the rename after redirection problem in this pull request? Ok, we can keep the tests, but this commit is unrelated and should come in a separate pull request.